src/components/collapsible-motion/collapsible-motion.js

```diff
@@ -26,25 +26,22 @@ export class ToggleAnimation extends React.Component {
   };
 
   static getDerivedStateFromProps(props, state) {
-    const animationName = props.isOpen
-      ? createOpeningAnimation(state.fullHeight)
-      : createClosingAnimation(state.fullHeight);
-    const animation = `${animationName} 200ms forwards`;
+    let animation = null;
 
-    let containerStyles = props.isOpen
+    const containerStyles = props.isOpen
       ? { height: 'auto' }
       : { height: 0, overflow: 'hidden' };
 
     if (props.isOpen !== state.isOpen) {
-      containerStyles = {
-        ...containerStyles,
-        animation,
-      };
+      animation = props.isOpen
```

Comment:
We could make `animation` a `const` now that we can determine it by a ternary.

Reply:
it would be a nested ternary though?

```js

    const animation =
      props.isOpen !== state.isOpen
        ? props.isOpen
          ? createOpeningAnimation(state.fullHeight)
          : createClosingAnimation(state.fullHeight)
        : null;

```

Reply:
...or a do expresso